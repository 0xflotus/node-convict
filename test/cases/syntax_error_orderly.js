exports.conf = {
  foo: {
    bar: 'string ["foo", "bar"' // note, the enum values are unclosed
  }
};
