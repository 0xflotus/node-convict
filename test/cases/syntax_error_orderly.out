'foo.bar' has an invalid format: Error: Parse error on line 1:
string ["foo", "bar"
--------------------^
Expecting ']', ',', got 'EOF'
